mir_eval/key.py

```diff
@@ -89,13 +91,13 @@ def weighted_score(reference_key, estimated_key):
     +------------------------------------------------------+-------+
     | Relationship                                         | Score |
     +------------------------------------------------------+-------+
-    | Same key                                             | 1.0   |
+    | Same key and mode                                    | 1.0   |
     +------------------------------------------------------+-------+
     | Estimated key is a perfect fifth above reference key | 0.5   |
     +------------------------------------------------------+-------+
-    | Relative major/minor                                 | 0.3   |
+    | Relative major/minor (same key signature)            | 0.3   |
     +------------------------------------------------------+-------+
-    | Parallel major/minor                                 | 0.2   |
+    | Parallel major/minor (same Key)                      | 0.2   |
```

Comment:
Key -> key ?

Reply:
the problem is that the key is the tonic note + the modality. However the tonic note is also called key. How do I distinguish between them?

Reply:
My idea is, for example, to use `Key` when I am referring to the full key (tonic + mode) and 'key' when it is used as a tonic note. What do you think @lostalen?

Reply:
BTW in that case is with k lower. You are right! Thank you!

Reply:
done in 
07a0d1d